syntax = "proto3";

package Hello;

service Greeter {
	rpc Hello (HelloRequest) returns (HelloResponse) {}
	rpc GetValue (KeyRequest) returns (KeyResponse) {}
	rpc SetValue (SetKeyRequest) returns(KeyResponse) {}
}

message HelloRequest {
	string name = 1;
}

message HelloResponse {
	string response = 1;
}

message SetKeyRequest {
    string key = 1;
    string value = 2;
}

message KeyRequest {
    string key = 1;
}

message KeyResponse {
    string value = 1;
}
